packages/drive-s3/src/AmazonWebServicesS3Storage.ts

```diff
@@ -147,14 +171,11 @@ export class AmazonWebServicesS3Storage extends Storage {
 		location = this._fullPath(location);
 
 		const params = { Key: location, Bucket: this.$bucket };
-
+		const result = await this.$driver.send(new GetObjectCommand(params));
+		const body = result.Body as NodeJS.ReadableStream;
```

Comment:
This should probably go in to the `try...catch` block

Reply:
Yep! fixed